src/gui/DatabaseWidget.cpp

```diff
@@ -259,8 +259,9 @@ bool DatabaseWidget::isEditWidgetModified() const
 {
     if (currentWidget() == m_editEntryWidget) {
         return m_editEntryWidget->hasBeenModified();
-    }
-    else {
+    } else if (currentWidget() == m_editGroupWidget) {
```

Comment:
Same as above, since you have an immediately preceding `return`, you don't need the `else` here.